config/cfme/manifest_5_11.json

```diff
@@ -214,7 +211,6 @@
         "uncommitted": null,
         "storage_domain_type": null,
         "host_storages": {
-          "ems_ref": null,
```

Comment:
Hey @pemcg so on 5.11 this is actually needed, there is no 5.12 manifest because the sync feature was removed
The column was removed on master but is still there on 5.12